src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="Stars stars--0">
```

Comment:
Fix classes everywhere 
```suggestion
      <div class="stars stars--0">
```